pywinauto/keyboard.py

```diff
@@ -605,23 +623,26 @@ def parse_keys(string,
             while modifiers:
                 if DEBUG:
                     print("MODS-", modifiers)
-                keys.append(VirtualKeyAction(modifiers.pop(), down = False))
+                keys.append(VirtualKeyAction(modifiers.pop(), down=False))
 
         # just in case there were any modifiers left pressed - release them
         while modifiers:
-            keys.append(VirtualKeyAction(modifiers.pop(), down = False))
+            keys.append(VirtualKeyAction(modifiers.pop(), down=False))
 
         return keys
 
+
     def LoByte(val):
         """Return the low byte of the value"""
         return val & 0xff
 
+
     def HiByte(val):
         """Return the high byte of the value"""
         return (val & 0xff00) >> 8
 
-    def SendKeys(keys,
+
+    def send_keys(keys,

```

Comment:
Calls to `SendKeys` should be replaced by `send_keys` also to avoid unnecessary DeprecationWarnings.